src/ate/test_programs/dut_lib/dut_lib.cc

```diff
@@ -60,5 +67,26 @@ void DutLib::DutConsoleTx(std::string& msg) {
   OtLibConsoleTx(transport_, msg.c_str());
 }
 
+void DutLib::DutTxCpProvisioningData(std::string& was,
```

Comment:
it seems like all of these could be `const std::string*`

Reply:
fixed.